.travis.yml

```diff
@@ -35,3 +35,11 @@ deploy:
       repo: alpheios-project/alpheios-core
       all_branches: true
       tags: true
+jobs:
+  include:
+    - script: npm run test-client-adapters
+    - script: npm run test-inflection-tables
+    - script: npm run test-components
+    - script: npm run test-data-models
+    # - script: npm run test-res-client
+    - script: npm run test-wordlist
```

Comment:
>I was to switch off res-client tests - because for some resons jest failed down after all 4 tests were completely done. I believe that this repo would be anyway refactored - so I switched them off for now.

While I was putting screenshot here - I found a misspell inside in res-client package.json
now it is tested well.
But there are two new problems, I believe from treebank changes

Reply:
Oh, I see, that's because the tests work failing. Sorry for missing that comment.

Reply:
> But there are two new problems, I believe from treebank changes

What are those tests that are failing? Tests in `res-client` works fine for me. If something fails, we can probably fix it in a separate PR.